bin/stashcp

```diff
@@ -141,6 +143,19 @@ def doStashCpSingle(sourceFile, destination, cache, debug=False):
         es_send(payload)
 
 
+def parse_job_ad():
+    temp_list = {}
+    try:
+        with open('.job.ad') as job_file:
```

Comment:
Why did you change this from reading in the environment?  I think reading in the environment is the best option.